server/http.go

```diff
@@ -67,35 +68,36 @@ func NewHTTPCache(cache cache.Cache, accessLogger cache.Logger, errorLogger cach
 }
 
 // Parse cache artifact information from the request URL
```

Comment:
Since we'd be returning two different strings, I think we should document what they are here now.